sphinx/search/stopwords/chinese_stopwords.txt

```diff
@@ -0,0 +1,2043 @@
+$
```

Comment:
The words with their length equal to 1 are already filtered by `word_filter()` method. Are these entry really needed?